SEED-platform/seed
# 2134 blank col setting 261

--------This PR contains a migration-------

#### Any background context you want to provide?
See tagged issue.

#### What's this PR do?
See commits:
- Column model has new recognize_empty bool field defaulted to False
- Merging records considers whether fields have recognize_empty=True
- Column model's recognize_empty added to serializer (now available to API)
- Org column settings page has Recognize Empty setting

Also:
- update Travis Postgres port

#### How should this be manually tested?
Test on production DB*

Toggle the Recognize Empty column setting. Import records that trigger merges and with Not Available or empty values in a particular field. When the setting is checked, see that empty values (truly blank, missing, or "Not Available") are treated like any other non-empty value during a merge.

**Notable caveat**: On import, if a known (by SEED) column is not provided/mapped to, SEED will think the incoming records have "empty" values for that column. This means, if recognize_blank is set to True for a particular column, and that column is not mapped to during import, any resulting merged records will have an empty value for that column/attribute.

Also, I suggest reading the **Note about production vs develop implementation** in the tagged issue.

#### What are the relevant tickets?
#2134 

#### Screenshots (if appropriate)
<img width="954" alt="Screen Shot 2020-03-06 at 4 51 19 PM" src="https://user-images.githubusercontent.com/30608004/76131364-6b372f80-5fcb-11ea-852f-bb11c9ff21b0.png">
